app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -114,12 +105,20 @@ export default class DiscoverySidebar extends React.Component {
     return moment(created_at).format("YYYY-MM-DD");
   }
 
-  static processSamples(newSamples, genomes) {
+  // TODO (gdingle): humanize better into M such as 34M
```

Comment:
is there an existing library for this? I'd like to be consistent

Reply:
We use the helper function `numberWithCommas` in some places (in `helper/string`), but the `toLocaleString` that you use seems like a better choice.

I would recommend changing `numberWithCommas` to use `toLocalString()`, renaming `numberWithCommas` to something like `formatNumber`, and writing a comment to explain when it's appropriate to use that function. And then use that function here. I don't think we need to humanize `100,000,000` to `100M` at the moment, and I think we're unlikely to have numbers bigger than that.

Reply:
Thanks for the background. I noted the `34M` style numbers because that's what was in the mock by @happyimadesignr . Not sure whether it's worth breaking consistency. 

As for `toLocaleString`, it does seem better in the long run as we serve a global audience. (In French for example, numbers are written such as `1 000 000,1`.

Do you want to see that change in this PR or a new one or file it away in Jira ?